ee/tasks/send_license_usage.py

```diff
@@ -0,0 +1,42 @@
+import posthoganalytics
+import requests
+from dateutil.relativedelta import relativedelta
+from django.utils import timezone
+
+from ee.clickhouse.client import sync_execute
+from ee.models.license import License
+from posthog.models import User
+
+
+def send_license_usage():
+    license = License.objects.first_valid()
```

Comment:
Should this be `first_valid`? Maybe we report on all valid licenses?

Reply:
Hm so everything else that we do (like checking what plan a user has access to) is all done on first_valid as well so this feels right. People should really only have one valid License enabled anyway.